configure.ac

```diff
@@ -415,9 +415,10 @@ AC_ARG_ENABLE(fast,
                    MPICHLIB_FFLAGS, and MPICHLIB_FCFLAGS. (default is -O2)
         ndebug   - Appends -DNDEBUG to MPICHLIB_CFLAGS.
         no-strict-alignment - relax strict alignment requirement
-        all|yes  - "O2" and "ndebug" are enabled
+        inline   - Force inline performance critical routines
+        all|yes  - "O2", "ndebug", and "force-inline" are enabled
         none     - None of above options, i.e. --disable-fast
-],,enable_fast=O2)
+],,[enable_fast=O2,inline])
```

Comment:
`--enable-fast=inline` is misleading. What we mean is `--enable-fast=always_inline`, and I think we should name it as what we mean.

I think I agree with @shintaro-iwasaki that it is questionable that we want to turn on "always inline" by default. 

Also, it is not obvious to me how do we *turn off* always inline? That is my reason that I prefer always inline turned off by default and `--enable-fast=all` or `--enable-fast=alwaysinline` turns it on.